setup.py

```diff
@@ -70,16 +70,16 @@ def run(self):
 }
 
 setup(
-    name="tf2onnx",
+    name='tf2onnx',
     version=VersionInfo.version,
     description='Tensorflow to ONNX converter',
     setup_requires=['pytest-runner'],
     tests_require=['graphviz', 'parameterized', 'pytest', 'pytest-cov', 'pyyaml'],
     cmdclass=cmdclass,
     packages=find_packages(),
     license='Apache License v2.0',
-    author='onnx@microsoft.com',
-    author_email='onnx@microsoft.com',
+    author='ONNX',
+    author_email='onnx-technical-discuss@lists.lfai.foundation',
```

Comment:
In https://lists.lfaidata.foundation/g/onnx-technical-discuss, I saw the email should be something like below:

onnx-technical-discuss@lists.lfaidata.foundation

Could you please confirm the domain is "lfai" or "lfaidata"?

Reply:
Thanks for catching. Done here. I will also update in onnx/onnx.